specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-04-15/cdn.json

```diff
@@ -3230,6 +3231,109 @@
         }
       }
     },
+    "DeliveryRuleUrlSigningAction": {
+      "description": "Defines the Url Signing action for the delivery rule.",
+      "x-ms-discriminator-value": "UrlSigning",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeliveryRuleAction"
+        }
+      ],
+      "required": [
+        "parameters"
+      ],
+      "properties": {
+        "parameters": {
+          "description": "Defines the parameters for the action.",
+          "$ref": "#/definitions/UrlSigningParameters"
+        }
+      }
+    },
+    "UrlSigningParameters": {
+      "description": "Defines the parameters for the Url Signing action.",
+      "required": [
+        "keys",
+        "hmacAlgorithm",
+        "queryStringParams",
```

Comment:
>queryStringParams [](start = 9, length = 17)

ParameterNameOverride

Reply:
Resolved in new PR.

---
In reply to: [302753236](https://github.com/Azure/azure-rest-api-specs/pull/6438#discussion_r302753236) [](ancestors = 302753236)